.github/workflows/e2e-nightly-34x.yml

```diff
@@ -47,12 +47,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Notify Slack on failure
```

Comment:
Should we add a 'success' block to this workflow? Or, maybe it's present and I'm not seeing it?

Reply:
Oh there wasn't a success block for v0.34. I can easily add one though.

Reply:
I've added one now :+1: